PUML.h

```diff
@@ -346,6 +355,36 @@ class PUML
 		checkH5Err(H5Pclose(h5alist));
 	}
 
+        template<typename T>
+	void addData(const T* rawData, unsigned long dataSize, DataType type)
+	{
+		int rank = 0;
+		int procs = 1;
```

Comment:
there is numRanks declared L117.

Reply:
I also think so, but this is kind of consistent with the rest of the code.